.travis.yml

```diff
@@ -0,0 +1,21 @@
+notifications:
+  irc:
+    channels: "chat.freenode.net#firedrake"
+    skip_join: true
+    on_success: change
+    on_failure: always
+    template: "%{repository}#%{build_number} (%{branch} - %{commit} : %{author}): %{message} | %{build_url}"
+
+language: python
+python:
+  - "2.7"
+  - "3.4"
+
+before_install:
+  - pip install pytest
```

Comment:
Travis does come with pytest by default so I don't thik this will have an effect. Did you want to?

    pip install -U pytest